SPECS/sympy/sympy.spec

```diff
@@ -0,0 +1,507 @@
+# We used to build with Theano support.  However, we no longer have a compatible
+# version of Theano in Fedora.  If aesara is ever packaged for Fedora, we can
+# use it instead.
+
+# We are archful (see below), but there are no ELF objects in the binary RPM.
+%global debug_package %{nil}
+%global _description\
+SymPy aims to become a full-featured computer algebra system (CAS)\
+while keeping the code as simple as possible in order to be\
+comprehensible and easily extensible. SymPy is written entirely in\
+Python and does not require any external libraries.
+Summary:        A Python library for symbolic mathematics
+Name:           sympy
+Version:        1.11.1
+Release:        5%{?dist}
+# The project as a whole is BSD-3-Clause.
+# The files in sympy/parsing/latex are MIT.
+License:        BSD-3-Clause AND MIT
+Vendor:         Microsoft Corporation
+Distribution:   Mariner
+URL:            https://sympy.org/
+Source0:        https://github.com/%{name}/%{name}/archive/%{name}-%{version}.tar.gz
+# Skip tests that require a display
+Patch0:         %{name}-circuitplot.patch
+BuildRequires:  %{py3_dist cython}
+BuildRequires:  python3-mpmath
+# This package used to be noarch, and should still be noarch.  However, because
+# there is no JDK available on i686 anymore, the antlr4 package is also not
+# available on i686.  When we can stop building on i686 altogether, we can bring
+# this back.  In the meantime, we cannot claim to be noarch, because the i686
+# build is different from the other arches in lacking BuildRequires: antlr4.
+# BuildArch:      noarch
+BuildRequires:  gcc
+BuildRequires:  gcc-c++
+BuildRequires:  gcc-gfortran
+BuildRequires:  python3-devel
+BuildRequires:  python3-numpy-f2py
+BuildRequires:  python3-pip
+BuildRequires:  python3-wheel
+#Test
+BuildRequires:  xorg-x11-server-Xvfb
+
+%description %{_description}
+
+%package -n python3-%{name}
+Summary:        A Python3 library for symbolic mathematics
+License:        BSD-3-Clause AND MIT
+Requires:       python-mpmath
+Recommends:     %{py3_dist cython}
```

Comment:
I don't think our tooling actually uses `Recommends, Suggests` and others that are not `Requires` or `BuildRequires`